flow/packet-headers/rsvp.yaml

```diff
@@ -579,15 +579,43 @@ components:
             If OTG is unable to generate a value the default value must be used. 
```

Comment:
Replace OTG with 'T/the OTG implementation for all auto field descriptions.`

Reply:
Done.